commands/Owner/update.js

```diff
@@ -18,7 +18,12 @@ exports.run = async (client, message) => {
     var message = await message.channel.send(embed);
 
     // Execute the update script
-    let result = await execute("sh update.sh");
+	if(args[0] = "dev"){
+		branch = "Dev";
+	}else{
+		branch = "master";
+	}
```

Comment:
```suggestion
	if (args[0] === "dev") {
		branch = "Dev";
	} else {
		branch = "master";
	}
```

Fixes bug where you are assigning `args[0]` to dev which always returns a truthy value.